googlepay/src/main/java/com/adyen/checkout/googlepay/internal/ui/DefaultGooglePayDelegate.kt

```diff
@@ -154,7 +142,7 @@ internal class DefaultGooglePayDelegate(
         )
     }
 
-    @Deprecated("Deprecated in favor of startGooglePayScreen()", replaceWith = ReplaceWith("startGooglePayScreen()"))
+    @Deprecated("Deprecated in favor of onSubmit()", replaceWith = ReplaceWith("onSubmit()"))
```

Comment:
Do we need to deprecate internal functions?

Reply:
Not necessary, but then we know we can clean it up later